docs/import/add_layer.js

```diff
@@ -51,21 +44,34 @@ function processNewEeLayer(asset, type) {
                  ee.Dictionary.fromLists(properties, stats))
           .then((columns) => {
             const layer = {
-              'assetType': type,
-              'eeName': asset,
-              'colorFunction': {
-                'columns': columns,
-                'currentStyle': 2,
-                'colors': {},
+              ...createCommonColorFunctionFields(asset, type),
+              colorFunction: {
+                columns: columns,
+                currentStyle: 2,
+                lastByPropertyStyle: 0,
+                colors: {},
               },
-              'displayName': '',
-              'displayOnLoad': false,
             };
             return prependToTable(layer);
           });
   }
 }
 
+/**
+ * Create fields that all ee layers have.
+ * @param {string} asset
+ * @param {LayerType} type
+ * @return {Object}
+ */
+function createCommonColorFunctionFields(asset, type) {
```

Comment:
nit: this function name talks about color, but isn't really about color. Maybe "createCommonLayerFields"?